tools/setup.py

```diff
@@ -82,18 +73,11 @@ def check_directory(self, name):
         if os.path.isdir(name):
             self.color.print_pink(name + " directory found", True)
         else:
-            self.color.print_red(name +
-                                 " directory not found, cloning for you",
-                                 True)
-            os.system("git clone https://github.com/SCE-Development/" + name)
-            os.system("cd " + name)
-
-    def check_docker(self):
-        """
-        This method is used to check for docker software
-        """
-        self.check_installation("docker", "docker --version",
-                                "https://docs.docker.com/desktop/")
+            self.color.print_red(
+                name + " directory not found, cloning for you", True)
+            subprocess.check_call("git clone"
+                                  + "https://github.com/SCE-Development/"
```

Comment:
Add a space after git clone because it gave me an error
![image](https://user-images.githubusercontent.com/14855752/89388451-e73bbd80-d6b8-11ea-8fbb-96eb89dfe3c9.png)

Reply:
^^^